Args, Mono doesn't have [StartInfo.Environment](https://msdn.microsoft.com/en-us/library/system.diagnostics.processstartinfo.environment%28v=vs.110%29.aspx):

_src/xp.runner/Command.cs(117,28): error CS1061: Type `System.Diagnostics.ProcessStartInfo` does not contain a definition for `Environment` and no extension method `Environment` of type `System.Diagnostics.ProcessStartInfo` could be found. Are you missing an assembly reference?_ 

This is a .NET 4.6 feature

--- a/src/xp.runner/Command.cs
+++ b/src/xp.runner/Command.cs
@@ -114,8 +114,7 @@ namespace Xp.Runners
                 main,
                 string.Join(" ", ArgumentsFor(cmd).Select(args))
             );
-
-            Environment.SetEnvironmentVariable("XP_CMD", string.Format(
+            proc.StartInfo.EnvironmentVariables.Add("XP_CMD", string.Format(
                 "exe={0}&version={1}&model={2}&command={3}",
                 binary,
                 Assembly.GetExecutingAssembly().GetName().Version,

--- a/src/xp.runner/Command.cs
+++ b/src/xp.runner/Command.cs
@@ -114,7 +114,8 @@ namespace Xp.Runners
                 main,
                 string.Join(" ", ArgumentsFor(cmd).Select(args))
             );
-            proc.StartInfo.Environment.Add("XP_CMD", string.Format(
+
+            Environment.SetEnvironmentVariable("XP_CMD", string.Format(
                 "exe={0}&version={1}&model={2}&command={3}",
                 binary,
                 Assembly.GetExecutingAssembly().GetName().Version,

